utils.py

```diff
@@ -3,9 +3,8 @@
 """
 import numpy as np
 import pandas as pd
-import matplotlib.pyplot as plt
-from scipy.stats import norm
-from statsmodels.distributions.empirical_distribution import ECDF
+
+import pints
 
 
 def run_replicates(iterations, n_replicates, test):
```

Comment:
Do we still need this one after adding `run_replicates2`?

Reply:
I'm happy to rewrite the other tests to all use `run_replicates`, if you like

Reply:
Yes, aren't all methods (e.g. `run_replicates_distance`, `run_replicates_annulus`) now handled by your method? As in, I'm not sure I see what rewriting would need to be done?

Reply:
I wasn't sure if that was the case so I didn't replace the other _x methods yet. Doing a PR for that now though!